src/site/content/en/blog/share-like-a-native/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Share like a native app
+subhead: Web apps can use the same system provided share capabilities as native apps.
+authors:
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
+alt: A hand sharing a web.dev article.
+description: |
+  With the Web Share API, web apps are able to use the same system provided
+  share capabilities as native apps. The Web Share API makes it possible for web
+  apps to share links, text, and files to other apps installed on the device in
+  the same way as native apps.
+tags:
+  - post
+  - capabilities
+---
+
+With the Web Share API, web apps are able to use the same system provided share
+capabilities as native apps. The Web Share API makes it possible for web apps to
+share links, text, and files to other apps installed on the device in the same
+way as native apps.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./wst-send.png" style="max-width: 25vw;" alt="System-level share target picker with an installed PWA as an option."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    System-level share target picker with an installed PWA as an option
+  </figcaption>
+</figure>
+
+{% Aside %}
+  Sharing is only half of the magic. Web apps can also be share
+  targets meaning they can receive data, links, text, and files from other
+  native or web apps. See the [Recieve shared
+  data](https://web.dev/receive-shared-data/) for details on how to register
+  as a share target.
+{% endAside %}
+
+## Concepts and usage
+
+### Capabilities and limitations
+
+Web share has the following capabilities and limitations:
+* It can only be used on a site that supports HTTPS.
+* It must be invoked in response to a user action such as a click. Invoking it
+  through the `onload` handler is impossible.
+* It can share, URLs, text, or files.
+
+### Sharing links and text
+
+To share links and text use the `share()` method, which is a promise-based
+method with a required properties object. The object must contain at least one
+of `title`, `text`, `url` or `files` or the browser throws a `TypeError`. You
+could for example share text without a url or vice versa. Allowing all three
+members expands the flexibility of use cases. Imagine if after running the code
+below, the user chose an email application as the target. The `title` parameter
+might become the email subject, the `text`, the message body, and the files, the
+attachments.
```

Comment:
I know parallelism makes the elisions here acceptable, but it also introduces syntactic ambiguity--took me a second read to realize this isn't a list. Might be worth including the omitted text: "The `title` parameter might become the email subject, `text` might become the message body, and `files` might become the attachments."